bin/update-extension-files.js

```diff
@@ -0,0 +1,340 @@
+/**
+ * External dependencies
+ */
+const { exec } = require( 'child_process' );
+const fs = require( 'fs' );
+const { getPluginsList, getThemesList } = require( 'wporg-api-client' );
+const axios = require( 'axios' );
+
+class UpdateExtensionFiles {
+	/**
+	 * Construct method.
+	 */
+	constructor() {
+		( async () => {
+			this.plugins = [];
+			this.themes = [];
+
+			await this.fetchData();
+			await this.storeData();
+		} )();
+	}
+
+	/**
+	 * Fetch AMP compatible plugins and themes.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async fetchData() {
+		let totalPage = 1;
```

Comment:
This will apparently fix the LGTM complaint:

```suggestion
		let totalPage;
```